gramex/gramex.yaml

```diff
@@ -216,6 +216,8 @@ log:
 eventlog:
     path: $GRAMEXDATA/logs/events.db          # Location of application event logs.
 
+eslog:
+    enabled: True
```

Comment:
This will automatically call `eslog(enabled=True)` in `gramex/services/__init__.py`. You can perform all initialization there.